client/js/components/statement/assessmentTable/ItemRow.vue

```diff
@@ -10,27 +10,49 @@
 <template>
   <div
     class="layout--flush u-ph-0_5"
-    :class="{'border--bottom': borderBottom, 'border--right': borderRight}">
+    :class="{
+      'border--bottom': borderBottom,
+      'border--right': borderRight,
+      'fullscreen': isFullscreen,
+      'fullscreen-row': isFullscreenRow
+    }">
     <div
       class="layout__item u-pl-0 c-at-item__row-icon color--grey"
       :title="Translator.trans(title)">
       <i
+        aria-hidden="true"
         class="fa"
         :class="icon"
-        aria-hidden="true"
         v-if="icon" />
```

Comment:
`v-if` should be ordered first.

Reply:
true a6016f86d8a3019e76cee541ddd8893ab99fe024